app/views/courses/entry_requirements/new.html.erb

```diff
@@ -30,7 +30,8 @@
     <%= form_with model: course,
                   url: continue_provider_recruitment_cycle_courses_entry_requirements_path(
                     @provider.provider_code,
-                    @provider.recruitment_cycle_year
+                    @provider.recruitment_cycle_year,
+                    course: @course_creation_params
```

Comment:
wonder how we could make this part a bit nicer to avoid having to constantly put `course: @course_creation_params` in every page, not sure if we can though